src/plugins/1/essentials/getting-started/index.md

```diff
@@ -34,87 +20,28 @@ git clone https://github.com/kuzzleio/kuzzle-core-plugin-boilerplate
-
-The following properties can be defined in this `manifest.json` file:
-
-* `name` (**required**): plugin unique identifier. Names can only contain lowercase letters, numbers, hyphens and underscores. 
-* `kuzzleVersion`: a non-empty string describing a [semver range](https://www.npmjs.com/package/semver#ranges), limiting the range of Kuzzle versions supported by this plugin. If not set, a warning is displayed on the console, and Kuzzle assumes that the plugin is only compatible with Kuzzle v1.x
-
-{{{deprecated "1.5.0"}}} Kuzzle still allows plugins to be loaded without a <code>manifest.json</code> file, for backward compatibility reasons, falling back to the <a href=https://docs.npmjs.com/files/package.json#name>package.json</a> file to retrieve the plugin's name. This will change in next major releases of Kuzzle.
-
----
+Since the `init` function is the very first to be called, it's loading the `configuration` and the `context`.
+You can now write your own functions and your own routes as described inside the `index.js`. You can also write unit tests if you're a see `steps.js`.
 
-## init function
-
-Plugins must expose an `init` function. If it is missing, Kuzzle fails to load the plugin and aborts its start sequence.
-
-The `init` method is called once during startup, and it is used to initialize the plugin.
-
-### Arguments
-
-```js
-init (config, context)
-```
-
-<br/>
-
-| Arguments | Type | Description |
-|-----------|------|-------------|
-| `config` | <pre>object</pre> | Contains the custom plugin configuration (see the [configuration](#configuration-default) chapter) |
-| `context` | <pre>object</pre> | The plugin context, exposing various accessors, constructors, and helpers. The other sections of this documentation detail the interfaces made available by this object |
-
-### Return
-
-The `init` function can optionally return a promise. If it does, Kuzzle waits for the promise to be resolved before continuing its own initialization.
-
-If a promise is returned, it must be resolved within the configured timeout (see `plugins.common.initTimeout` in Kuzzle's [configuration]({{ site_base_path }}guide/1/essentials/configuration/))
-
-If a promise is returned and rejected, or if the `init` function throws an error, Kuzzle aborts its start sequence and shuts down.
-
----
-
-## Configuration
-
-When Kuzzle calls the plugin `init` method, it passes the plugin's custom configuration to it.
-
-Custom configuration parameters are specified for each plugin in the `plugins` object of the Kuzzle [configuration file]({{ site_base_path }}guide/1/essentials/configuration). 
-
-For example:
-
-```json
-{
-  "plugins": {
-    "foobar-plugin": {
-      "option_1": "option_value",
-      "option_2": "option_value"
-    }
-  }
-}
-```
+<div class="alert alert-info">
+You can find more information about the <code>init</code> function <a href="https://docs.kuzzle.io/plugins/1/plugin-layout/init-function/"> here</a>.
```

Comment:
dead link

Reply:
Normal, the link will work when the PR will be merged :p